### PR TITLE
[ast] Update D2S review comments

### PR DESCRIPTION
**Pull request by Jacob-Levy**
One file, `ast.sv,`  is updated with the review comments.

@arnonsha @zi-v 
Signed-off-by: Jacob Levy <jacob.levy@nuvoton.com>

### CODE REVIEW

File: hw/top_earlgrey/ip/ast/rtl/ast.sv [hunk 1]
```diff
@@ -437,7 +437,7 @@ logic clk_aon_ext;
 assign clk_aon_ext = clk_osc_byp_i.aon;
 `endif
 
-assign rst_aon_clk_n = vcc_pok_str;
+assign rst_aon_clk_n = vcc_pok_str && vcaon_pok;
```
Comment (marnovandermaas) [inline hw/top_earlgrey/ip/ast/rtl/ast.sv:440]: Why did you add this? Would you be able to explain this in a short comment?

**Review by marnovandermaas — COMMENTED** [review]
Not really my area, but I added a question.

File: hw/top_earlgrey/ip/ast/rtl/ast.sv (hunk 1 above)
Comment (Jacob-Levy) [reply hw/top_earlgrey/ip/ast/rtl/ast.sv:440]: Power simulation found an issue between VCC & AON.

Comment (marnovandermaas) [reply hw/top_earlgrey/ip/ast/rtl/ast.sv:440]: Ok, thanks for the explanation! Do you think it's worth adding a comment in the code?

Comment (Jacob-Levy) [reply hw/top_earlgrey/ip/ast/rtl/ast.sv:440]: Thanks! No need to add comments on fixed issue (GITLAB capture all the information).

Comment (marnovandermaas) [reply hw/top_earlgrey/ip/ast/rtl/ast.sv:440]: Ok, sounds good.

**Comment by tjaychen** [comment]
thakns for helping with reviews @marnovandermaas 
@Jacob-Levy when it's ready to merge, you can use the "ready-to-merge" label, then someone with merge rights will push this in.

**Comment by Jacob-Levy** [comment]
@tjaychen I do not have merge privileges.... 
In any case someone need to approve it first.

**Comment by tjaychen** [comment]
there is a "label" called ready-to-merge. It basically let people with
merge rights know that something can be pushed in.
That way you don't have to rely on 1 or 2 people to do it all the time,
anyone with merge rights can handle it.

**Comment by Jacob-Levy** [comment]
@tjaychen I know, but I am not privie to that action since I am not the "code owner."..

**Comment by Jacob-Levy** [comment]
@tjaychen Can you approve it?

**Comment by msfschaffner** [comment]
> @tjaychen I know, but I am not privie to that action since I am not the "code owner."..

Adding that label should not require code owner priviledges. You can just select it in the drop-down in the `Labels` section on the right hand side of this page.

**Review by msfschaffner — APPROVED** [review]

**Comment by Jacob-Levy** [comment]
@tjaychen @msfschaffner 
In order to be able to use the the Label, `Ready-to-Merge`, I need to be a member of [lowRISC/opentitan-committers] which I am not :(
```
[Jacob-Levy](https://github.com/Jacob-Levy)
[Jacob](https://github.com/Jacob-Levy)
Opened this pull request
Committed to this repository in the past week
Member of [lowRISC/opentitan](https://github.com/orgs/lowRISC/teams/opentitan)
Member of [lowRISC](https://github.com/lowRISC)
```
